### PR TITLE
Refactoring/types, files, renamings

### PR DESCRIPTION
**Pull request by CiprianDraghici**

### CODE REVIEW

File: src/reduxStore/slices/toastsSlice.ts [hunk 1]
```diff
@@ -1,13 +1,13 @@
 import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { TRANSACTION_STATUS_TOAST_ID } from 'constants/transaction-status';
```
Comment (arhtudormorar) [inline src/reduxStore/slices/toastsSlice.ts:2]: constants/transactionStatus

File: src/types/index.ts [hunk 2]
```diff
@@ -15,10 +15,10 @@ export interface TokenLoginType {
   signature?: string;
 }
 
-export * from './transactions';
+export * from './transactions.types';
 export * from './server-transactions';
```
Comment (arhtudormorar) [inline src/types/index.ts:19]: serverTransactions.types.ts

File: src/types/index.ts [hunk 3]
```diff
@@ -15,10 +15,10 @@ export interface TokenLoginType {
...
-export * from './enums';
-export * from './account';
-export * from './network';
-export * from './dapp-provider';
-export * from './login';
+export * from './enums.types';
+export * from './account.types';
+export * from './network.types';
+export * from './dapp-provider.types';
```
Comment (arhtudormorar) [inline src/types/index.ts:23]: dapp-provider.types.ts --> dappProvider.types.ts

File: src/UI/TransactionsTable/TransactionIcon.tsx [hunk 4]
```diff
@@ -9,8 +9,8 @@ import globalStyles from 'assets/sass/main.scss';
 import { getOperationsMessages } from 'components/TransactionsInterpreter/helpers/getOperationsMessages';
 import { getReceiptMessage } from 'components/TransactionsInterpreter/helpers/getReceiptMessage';
 import getScResultsMessages from 'components/TransactionsInterpreter/helpers/getScResultsMessages';
-import { TransactionServerStatusesEnum } from 'types/enums';
-import { ServerTransactionType } from 'types/server-transactions';
+import { TransactionServerStatusesEnum } from 'types/enums.types';
+import { ServerTransactionType } from 'types/serverTransactions';
```
Comment (arhtudormorar) [inline src/UI/TransactionsTable/TransactionIcon.tsx:13]: serverTransactions.types

File: src/reduxStore/slices/toastsSlice.ts [hunk 5]
```diff
@@ -1,13 +1,13 @@
...
-import { logoutAction } from '../commonActions';
 import {
   CustomToastType,
   FailTransactionToastType,
   TransactionToastType
-} from 'types/toasts';
+} from 'types/toasts.types';
 import { getUnixTimestamp } from 'utils/dateTime/getUnixTimestamp';
-import { TRANSACTION_STATUS_TOAST_ID } from 'constants/transaction-status';
+import { logoutAction } from '../commonActions';
```
Comment (arhtudormorar) [inline src/reduxStore/slices/toastsSlice.ts:10]: LOGOUT_ACTION

File: src/types/index.ts [hunk 6]
```diff
@@ -15,10 +15,10 @@ export interface TokenLoginType {
...
 
-export * from './transactions';
-export * from './server-transactions';
-export * from './enums';
-export * from './account';
-export * from './network';
-export * from './dapp-provider';
-export * from './login';
+export * from './transactions.types';
+export * from './serverTransactions';
```
Comment (arhtudormorar) [inline src/types/index.ts:19]: serverTransactions.types

File: src/reduxStore/slices/toastsSlice.ts (hunk 5 above)
Comment (CiprianDraghici) [reply src/reduxStore/slices/toastsSlice.ts:10]: this is not a constant, it's an object created with @reduxjs/toolkit

**Review by arhtudormorar — APPROVED** [review]

**Pull request merged**